ui/app/routes/auth/index.ts

```diff
@@ -0,0 +1,28 @@
+import ApiService from 'waypoint/services/api';
+import { Empty } from 'google-protobuf/google/protobuf/empty_pb';
+import { ListOIDCAuthMethodsResponse } from 'waypoint-pb';
+import Route from '@ember/routing/route';
+import SessionService from 'ember-simple-auth/services/session';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class AuthIndex extends Route {
+  @service session!: SessionService;
+  @service api!: ApiService;
+  @tracked authMethods!: ListOIDCAuthMethodsResponse;
+
+  async beforeModel(): Promise<void> {
+    let authMethods = await this.api.client.listOIDCAuthMethods(new Empty(), this.api.WithMeta());
+    this.authMethods = authMethods;
+    return;
+  }
```

Comment:
Why do we do this in `beforeModel` rather than directly in `model`?

Reply:
I think that's a letfover from the previous prototype, where I was thinking about doing a redirect to an `auth/oidc` route, I'll move everything to  model as this is not happening anymore.